pkgs/applications/networking/gotify-client/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  version = "2.1.1";
+  name = "gotify-client-${version}";
```

Comment:
Please consider using ```pname```.

Reply:
That doesn't fit here IMO because it won't give any benefit - no other variables will be using it.

Reply:
Most of the bots and scripts are using ```pname``` variable. It's considered a standard practice and lot of effort has been put into migrating other packages over to pname structure.

Reply:
:+1: